freedomofpress/pressfreedomtracker.us
# Track Matomo Events on Filter

closes #1599 

Adds specific filter tracking to matomo by [sending events](https://developer.matomo.org/guides/tracking-javascript-guide#manually-trigger-events).

## Testing
- Set `ANALYTICS_ENABLED` to `True` in [base.py](https://github.com/freedomofpress/pressfreedomtracker.us/blob/develop/tracker/settings/base.py#L219) and run the app
- Open the [incidents database locally](http://localhost:8000/all-incidents/) and set a few filters
- The network tab should show the tracking requests
